src/views/Proposals.vue

```diff
@@ -0,0 +1,118 @@
+<template>
```

Comment:
Why do we need to create this file for the mobile nav?

Reply:
No not just for mobile, now that we are using `router-view` we need separate views for `Proposals` and `About`.

Reply:
Oh, that's cool. I've never used a second router-view like that.